test/cmdline_test.py

```diff
@@ -219,6 +221,19 @@ def test_subclasses_2(self):
         self.assertEquals(task.x, 'xyz')
 
 
+@mock.patch('luigi.cmdline.luigi.interface')
+class LuigiMainTestCase(unittest.TestCase):
+    def test__main__(self, interface_mock):
+        luigi_main_path = os.path.join(os.path.dirname(luigi.__file__),
+                                       '__main__.py')
+
+        # import as not __main__ module
+        imp.load_source('ololo', luigi_main_path)
```

Comment:
`ololo` means import module not as '__main__'.

I can not imagine that situation and added test for the sake of completeness.